README.md

```diff
@@ -6,70 +6,54 @@
 
 # Data is Better Together
```

Comment:
@davanstrien should we mention people can apply to HF grants for projects like these to together further the development of high-quality open-source datasets on the Hugging Face Hub?

Reply:
Good idea. Maybe something like: 

"If you are working on a valuable community developed dataset but are constrained by the resources you have available get in touch with us and we can see if there are ways we can support your work."